_docs/install/prerequisites/clusterdiscovery.md

```diff
@@ -75,4 +77,17 @@ When multiple methods are provided, the arguments are evaluated and attempted in
 JOIN=d53e9fae-7436-4185-82ea-c0446a52e2cd,172.28.128.3,172.28.128.9
 ```
 
-* [Checking the cluster status]({%link _docs/install/health.md %})
+### Speacial Case: One-node clusters
+
+Some deployments require the use of StorageOS in a one-node cluster setup.
+It is important to remember that even in these setups the JOIN keyword must be populated.
```

Comment:
Merge the two sentences to:

The `JOIN` command line argument is always required, even in clusters with only one node.